src/shopDetail/ShopDetail.tsx

```diff
@@ -95,6 +95,17 @@ export const ShopDetail = ({ route }: TProps) => {
 
   return (
     <>
+      <Analytics
+        contexts={{
+          reward: {
+            id: reward.id,
```

Comment:
is er een reden waarom reward niet destructured is?

Reply:
Geen idee & ook niet echt in scope van deze PR; Ik heb de implementatie van deze component niet veranderd.